agents/components/MemberInvites.js

```diff
@@ -1,17 +1,27 @@
 import PropTypes from 'prop-types'
 import React from 'react'
+import { connect as connectRedux } from 'react-redux'
 import { connect as connectFela } from 'react-fela'
-import { Field, FieldArray, reduxForm as connectForm } from 'redux-form'
+import { Field, FieldArray, formValueSelector, reduxForm as connectForm } from 'redux-form'
 import { flow } from 'lodash'
 import { TextField } from 'redux-form-material-ui'
 
 import styles from '../styles/MemberInvites'
 import Button from '../../app/components/Button'
 
-function renderMembers ({ fields, meta: { error, submitFailed } }) {
+function renderMembers ({ fields, meta: { error, submitFailed }, formProps }) {
```

Comment:
@ahdinosaur so this works as we expect, but there are a few things i'm not entirely sure/comfortable about:

- scrappy nested conditional logic to check that the last row has values in it
- react does not like the fact that this is taking place within the render cycle
  - i thought about retreating to using componentWillReceiveProps, but then was unsure how i would gain access to `fields` in order to push a new one

whaddya think?